flask_ansible.md

```diff
@@ -0,0 +1,182 @@
+# Installing Flask with Ansible on CentOS 7
+
+In this guide, we will be deploying a simple Flask application on a CentOS 7 droplet. We will use the provisioning tool Ansible to carry out this task, instead of running commands one by one. While building our deployment script, we will get the chance to explore a wide array of Ansible's features and also some best practices.
+
+## Prerequisites
+Before starting on this guide, you should have a non-root user configured on your server. The user will be responsible for setting up and keeping the website running (and it's not a good idea for a website to be running as root). This user needs to have sudo privileges so that it can perform administrative functions, such as installing packages. To learn how to set this up, follow our [initial server setup guide](https://www.digitalocean.com/community/tutorials/initial-server-setup-with-ubuntu-14-04) . 
+
+You also need to have ansible installed on your local machine, that is, the machine you use to ssh into your droplet. You can find installation instructions for all platforms in the [official Ansible documentation](http://docs.ansible.com/ansible/intro_installation.html).
+
+## Why Ansible
+
+Using a provisioning framework has the advantage of letting us do a one-command deployment where we don't need to interact with the terminal. Ansible in particular is built around the concept of *indempotency* - the capability to run the same deployment script again and again and to have the certainty that the system will always reach the desired state safely, without unnecessarily redoing work that has already been done or throwing errors when a command is executed twice. 
+
```

Comment:
perhaps here I should add something about how great it is to deploy deployments (!!) to multiple hosts from one central place that just has ssh access to them